iati/core/default.py

```diff
@@ -104,36 +105,94 @@ def codelists(version=None, bypass_cache=False):
+
+    Returns:
+        dict: Containing the version (as keys) and a corresponding ActivitySchema object (as values).
+
+    Todo:
+        Handle a `bypass_cache` parameter.
+    """
+    output = {}
+    for version in iati.core.constants.STANDARD_VERSIONS:
+        activity_schema_paths = iati.core.resources.get_all_activity_schema_paths(version)
+        output[version] = iati.core.ActivitySchema(activity_schema_paths[0])
+
+    return output
+
+
+def organisation_schemas():
+    """Return a dictionary of the default OrganisationSchema objects for all versions of the Standard.
+
+    Returns:
+        dict: Containing the version (as keys) and a corresponding OrganisationSchema object (as values).
+
+    Todo:
+        Handle a `bypass_cache` parameter.
+    """
+    output = {}
+    for version in iati.core.constants.STANDARD_VERSIONS:
+        organisation_schema_paths = iati.core.resources.get_all_organisation_schema_paths(version)
+        output[version] = iati.core.OrganisationSchema(organisation_schema_paths[0])
+
+    return output
+
+
 def schemas(bypass_cache=False):
-    """Locate the default Schemas.
+    """Locate all the default IATI Schemas and return them within a dictionary.
 
     Args:
         bypass_cache (bool): Whether the cache should be bypassed, instead reloading data from disk even if it's already been loaded.
 
     Returns:
-        dict: A dictionary containing all the Schemas for versions of the Standard. The version of the Standard is the key. An iati.core.Schema() is each value.
+        dict: A dictionary containing all the Schemas for versions of the Standard. This returns the name of the Schema (as the key) and a subclass of iati.core.schemas.Schema() (as the value).
 
     Warning:
         The `bypass_cache` parameter could potentially be implemented in a cleaner manner. It also shouldn't really exist until a clear use-case is defined - changes elsewhere in the library may make it redundant.
 
     Todo:
-        Allow creation of Schemas by XML rather than name.
+        Consider the Schema that defines the format of Codelists.
 
-        Handle the difference between Organisation and Activity Schemas.
+        Handle the `bypass_cache` parameter (and test a cache bypass where data is updated).
 
-        Consider the Schema that defines the format of Codelists.
+        Needs to handle multiple versions of the Schemas. This will probably involve passing in a version as a param, which should tidy up the function too.
+    """
+    schemas_by_type_and_version = {
+        'activity': activity_schemas(),
+        'organisation': organisation_schemas()
+    }
+
+    latest_version = iati.core.constants.STANDARD_VERSION_LATEST
+    for _, schemas_by_version in schemas_by_type_and_version.items():
+        schema_object = schemas_by_version[latest_version]
+        name = schema_object.source_path.split(os.sep).pop()[:-len(iati.core.resources.FILE_SCHEMA_EXTENSION)]
+        if (name not in _SCHEMAS.keys()):
```

Comment:
Are you suggesting removing the `bypass_cache` parameter, or removing this function altogether - perhaps just having `iati.core.default.schema` do something like:

```
def schema(name, version=None):
    try:
        return organisation_schemas()[version][name] if organisation_schemas()[version][name]
        return activity_schemas()[version][name]
```